### PR TITLE
Shows details of compilation info in SparkUI's engine tab

### PR DESCRIPTION
**Pull request by bowenliang123**
### _Why are the changes needed?_

Adding compilation details info in SparkUI's Engine tab
- shows details of kyuubi version, including revision, revision time and branch
- show compilation version of Spark, Scala, Hadoop and  Hive

image

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [x] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/master/contributing/code/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/spark/ui/EnginePage.scala [hunk 1]
```diff
@@ -58,6 +58,15 @@ case class EnginePage(parent: EngineTab) extends WebUIPage("") {
         <strong>Kyuubi Version: </strong>
         {KYUUBI_VERSION}
       </li>
+      <li>
+        <strong>Compilation Info:</strong>
```
Comment (pan3793) [inline externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/spark/ui/EnginePage.scala:62]: How about ?
```
Compilation Revision: ...
Compilation Branch: ...
Compilation with: Spark x.y.z, Hadoop x.y.z ...
```

Comment (bowenliang123) [reply externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/spark/ui/EnginePage.scala]: OK, Changed to 'Compilation Revision' and 'Compilation with'.

File: externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/spark/ui/EnginePage.scala [hunk 2]
```diff
@@ -84,10 +93,6 @@ case class EnginePage(parent: EngineTab) extends WebUIPage("") {
           <strong>Background execution pool threads active: </strong>
           {engine.backendService.sessionManager.getActiveCount}
         </li>
-          <li>
-            <strong>Background execution pool work queue size: </strong>
-            {engine.backendService.sessionManager.getWorkQueueSize}
```
Comment (cxzl25) [inline externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/spark/ui/EnginePage.scala:89]: why remove this

Comment (bowenliang123) [reply externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/spark/ui/EnginePage.scala]: Oop. Nice catch. Reverted.

**Review by ulysses-you — APPROVED** [review]

**Review by Yikf — APPROVED** [review]

**Review by pan3793 — APPROVED** [review]

**Review by cxzl25 — APPROVED** [review]

**Comment by bowenliang123** [comment]
Thanks, merged to master (1.8.0) and 1.7.2

**Pull request closed without merging**